explorer/src/providers/accounts/index.tsx

```diff
@@ -250,6 +250,11 @@ async function fetchAccountInfo(
                       metadata,
                       connection
                     );
+
+                    if (!editionInfo.masterEdition && !editionInfo.edition) {
+                      throw new Error("No edition found");
+                    }
```

Comment:
We already have UI to show that a `MasterEdition` does not exist. See `NFTHeader`

Reply:
I can definitely revert this. Metaplex deployed NFT metadata accounts on standard token mints causing tokens like SAMO (https://explorer.solana.com/address/7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU) to render as NFTs. Do you have a better fix in mind?

Reply:
Let me take a look.

Reply:
Much appreciated!

Reply:
@oJshua we could check for `decimals: 0` to confirm it's a Metaplex NFT. If there is Metaplex Metadata and `decimals` > 0. We could assume it's a non-NFT Token.

Reply:
"A Master Edition object can only be created for mints with supply of one and decimals of zero." Creators "should" have `MasterEditions` associated with NFTs, but it's too late to strictly enforce. I think decimal 0 check should be good enough for this.

Reply:
@oJshua https://github.com/solana-labs/solana/pull/21132